spec/requests/posts_spec.rb

```diff
@@ -3,14 +3,9 @@
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
     it 'returns http success' do
-      get '/posts/index'
-      expect(response).to have_http_status(:success)
-    end
-  end
-
-  describe 'GET /show' do
-    it 'returns http success' do
-      get '/posts/show'
+      # Assuming you have a user and some posts associated with that user
+      user = create(:user)
+      get user_posts_path(user)
       expect(response).to have_http_status(:success)
     end
```

Comment:
- [ ] Please make sure to check the below for each actions(index and show) for user and post controllers. Currently, you're checking only status. You should place the tests only in files re to posts and users, don't create other files for tests example `controllers_spec.rb`

![image](https://github.com/DevKaranJ/Blog/assets/83803414/c1c5b1d0-61e5-411c-94bf-22070ab5bae7)